sql/search_aggregates_v5.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TEMP FUNCTION
+  udf_mode_last(list ANY TYPE) AS ((
+    SELECT
+      _value
+    FROM
+      UNNEST(list) AS _value
+    WITH
+    OFFSET
+      AS
+    _offset
+    GROUP BY
+      _value
+    ORDER BY
+      COUNT(_value) DESC,
+      MAX(_offset) DESC
+    LIMIT
+      1 ));
+
+WITH
+  summary_addon_version AS (
```

Comment:
We haven't been using a search addon for tracking in quite some time (I think since Firefox 56 or 57), it might be worth asking whether anyone still wants to track this information. Mike Connor I think was the main person who cared.

Reply:
I think we agreed we would just take these out for now (can always put it back in if someone complains, which I very much doubt)